test/system/102-list.bats

```diff
@@ -102,3 +102,19 @@ teardown() {
 
     rm -f "$BATS_TMPDIR"/Containerfile
 }
+
+@test "list: List should not show two human readble names twice" {
+  pull_and_rename_default_image fedora 32
+
+  run podman images
```

Comment:
It's not necessary to `RUN podman images` to double check whether the image really showed up or not.  For example, we don't check whether `create_default_container` worked or not.

Reply:
Done.

Reply:
On second thoughts, there's this `list_images` function that we use to count the number of images.  We can use it here.